app/assets/stylesheets/donor.css.scss

```diff
@@ -0,0 +1,216 @@
+@mixin clear-list-margins() {
+  margin: 0;
+  list-style-type: none;
+}
+
+
+.profile-summary-donate-link {
+  background: $white;
+  border: 1px solid $donor-profile-border-grey;
+  border-radius: 4px;
+  color: $medium-green;
+  margin: 0 auto;
+  padding: 5px 10px;
+}
+
+.update-profile-link {
+  background: $donor-profile-light-grey;
+  color: $donor-profile-darker-grey;
+  padding: 8px;
+  text-decoration: none;
+  width: 100px;
+}
+
+.donor-dashboard-block {
+  @include full-width-block($off-white);
+  @include block-padding();
+  min-height: 900px;
+
+  .dashboard {
+    border: 1px solid $dark-grey;
+    @include drop-shadow(1);
+
+    .card-header {
+      @extend .text-center;
+      border-bottom: 1px solid $dark-grey;
+      background-color: $medium-green;
+      .card-header-title {
```

Comment:
Rule declaration should be preceded by an empty line<br>Selector should have depth of applicability no greater than 2, but was 4